frontend/src/app/components/accelerate-checkout/accelerate-checkout.component.ts

```diff
@@ -60,6 +60,7 @@ export class AccelerateCheckout implements OnInit, OnDestroy {
   @Input() eta: ETA;
   @Input() scrollEvent: boolean;
   @Input() cashappEnabled: boolean = true;
+  @Input() applePayEnabled: boolean = false;
```

Comment:
This input parameter doesn't seem to be used externally yet - do we want to enforce the same "acceleration eligibility criteria" as we do for cashapp payments?

see https://github.com/mempool/mempool/blob/82c271267a27678f71a519fb47259179944295e7/frontend/src/app/components/transaction/transaction.component.ts#L452-L454
https://github.com/mempool/mempool/blob/82c271267a27678f71a519fb47259179944295e7/frontend/src/app/components/tracker/tracker.component.ts#L756-L758